### DISCUSSION TITLE
I encountered a problem where images are not displaying when configuring the website.

### DISCUSSION THREAD

**Discussion by Tomp-Akoe** [Sped0n/bridget#429] [Q&A] [answered]
1. After completing the configuration, I found that the images could not be displayed. For details, please refer to the screenshot, where it is evident that the bottom right corner is unable to display photos. However, when deployed locally, the number of photos flickers briefly every time the page is refreshed.
![image](https://github.com/user-attachments/assets/adb2f35d-9c1c-4cd6-bc54-a04bcd5c8689)

2.After running hugo server, the display is as follows:
```
PS E:\tools\hugo_0.140.0_windows-amd64\myblog> hugo server
Watching for changes in E:\tools\hugo_0.140.0_windows-amd64\myblog\{archetypes,assets,content,data,i18n,layouts,package.json,static}
Watching for config changes in E:\tools\hugo_0.140.0_windows-amd64\myblog\go.mod
Start building sites …
hugo v0.140.0-3f35721fb2c75a1f7cc5a7a14400b66e73d4b06e+extended windows/amd64 BuildDate=2024-12-17T14:20:55Z VendorInfo=gohugoio

WARN  found no layout file for "html" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN  found no layout file for "html" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN  Raw HTML omitted while rendering "E:/tools/hugo_0.140.0_windows-amd64/myblog/content/Info/index.md"; see https://gohugo.io/getting-started/configuration-markup/#rendererunsafe
You can suppress this warning by adding the following to your site configuration:
ignoreLogs = ['warning-goldmark-raw-html']

                   | EN
-------------------+-----
  Pages            |  9
  Paginator pages  |  0
  Non-page files   |  1
  Static files     | 15
  Processed images |  0
  Aliases          |  0
  Cleaned          |  0

Built in 21 ms
Environment: "development"
Serving pages from disk
Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
Web Server is available at //localhost:1313/ (bind address 127.0.0.1)
Press Ctrl+C to stop
```

**Comment by 713b76b7-2247-4bd5-98a7-014226363f9b** [comment]
[Minimum reproduction](https://antfu.me/posts/why-reproductions-are-required) is needed

**Reply by Tomp-Akoe** [reply]
Okay, I will copy it to you tomorrow.

**Comment by Tomp-Akoe** [comment]
> [Minimum reproduction](https://antfu.me/posts/why-reproductions-are-required) is needed

Could you please provide more details about your requirements? For example, which folder or document, etc., so that I can assist you more specifically?

**Reply by 713b76b7-2247-4bd5-98a7-014226363f9b** [reply]
Maybe let's start with the directory structure? Like https://github.com/Sped0n/bridget/issues/388

Also, a copy of config would be nice

**Reply by Tomp-Akoe** [reply]
> Maybe let's start with the directory structure? Like https://github.com/Sped0n/bridget/issues/388
> 
> 
> 
> Also, a copy of config would be nice

Okay, I will copy it to you tomorrow. Thank you.

**Reply by Tomp-Akoe** [reply]
> 也许让我们从目录结构开始？赞 #388
> 
> 此外，复制 config 会很好

Hello, I have pulled a total of two pieces of content

1. The project directory of the theme: 
```
│  .eslintignore
│  .eslintrc.json
│  .gitignore
│  .hugo_build.lock
│  .prettierignore
│  .prettierrc.json
│  CODE_OF_CONDUCT.md
│  go.mod
│  LICENSE
│  package.json
│  pnpm-lock.yaml
│  README.md
│  theme.toml
│  tsconfig.json
│  vite.config.ts
│
├─.github
│  │  codeql-config.yml
│  │  dependabot.yml
│  │
│  ├─ISSUE_TEMPLATE
│  │      bug_report.yaml
│  │      config.yml
│  │      feature_request.yaml
│  │
│  └─workflows
│          build.yml
│          codeql.yml
│          eslint.yml
│
├─archetypes
│      default.md
│
├─assets
│  ├─scss
│  │  │  critical.scss
│  │  │  style.scss
│  │  │  _variables.scss
│  │  │
│  │  ├─_core
│  │  │      _base.scss
│  │  │      _font.scss
│  │  │      _mixins.scss
│  │  │      _reset.scss
│  │  │      _typography.scss
│  │  │
│  │  └─_partial
│  │          _article.scss
│  │          _collection.scss
│  │          _container.scss
│  │          _customCursor.scss
│  │          _gallery.scss
│  │          _nav.scss
│  │          _stage.scss
│  │          _stageNav.scss
│  │
│  └─ts
│      │  main.tsx
│      │  resources.ts
│      │  state.tsx
│      │  utils.ts
│      │
│      ├─desktop
│      │      customCursor.tsx
│      │      layout.tsx
│      │      nav.tsx
│      │      stage.tsx
│      │      stageNav.tsx
│      │
│      └─mobile
│              collection.tsx
│              gallery.tsx
│              galleryImage.tsx
│              galleryNav.tsx
│              layout.tsx
│
├─content
│  │  go.mod
│  │
│  ├─Erwitt
│  │      1.jpg
│  │      10.jpg
│  │      11.jpg
│  │      12.jpg
│  │      13.jpg
│  │      14.jpg
│  │      15.jpg
│  │      16.jpg
│  │      17.jpg
│  │      18.jpg
│  │      19.jpg
│  │      2.jpg
│  │      20.jpg
│  │      21.jpg
│  │      22.jpg
│  │      23.jpg
│  │      24.jpg
│  │      25.jpg
│  │      3.jpg
│  │      4.jpg
│  │      5.jpg
│  │      6.jpg
│  │      7.jpg
│  │      8.jpg
│  │      9.jpg
│  │      index.md
│  │
│  ├─Gruyaert
│  │      1.jpg
│  │      10.jpg
│  │      11.jpg
│  │      12.jpg
│  │      13.jpg
│  │      14.jpg
│  │      15.jpg
│  │      16.jpg
│  │      17.jpg
│  │      18.jpg
│  │      19.jpg
│  │      2.jpg
│  │      20.jpg
│  │      21.jpg
│  │      22.jpg
│  │      23.jpg
│  │      24.jpg
│  │      25.jpg
│  │      26.jpg
│  │      27.jpg
│  │      28.jpg
│  │      29.jpg
│  │      3.jpg
│  │      30.jpg
│  │      31.jpg
│  │      32.jpg
│  │      33.jpg
│  │      34.jpg
│  │      35.jpg
│  │      4.jpg
│  │      5.jpg
│  │      6.jpg
│  │      7.jpg
│  │      8.jpg
│  │      9.jpg
│  │      index.md
│  │
│  ├─Info
│  │      index.md
│  │
│  └─Webb
│          1.jpg
│          10.jpg
│          11.jpg
│          12.jpg
│          13.jpg
│          14.jpg
│          15.jpg
│          16.jpg
│          17.jpg
│          18.jpg
│          19.jpg
│          2.jpg
│          20.jpg
│          21.jpg
│          22.jpg
│          23.jpg
│          24.jpg
│          25.jpg
│          26.jpg
│          27.jpg
│          28.jpg
│          29.jpg
│          3.jpg
│          30.jpg
│          4.jpg
│          5.jpg
│          6.jpg
│          7.jpg
│          8.jpg
│          9.jpg
│          index.md
│
├─data
├─doc
│      getStarted.md
│
├─i18n
│      de.toml
│      en.toml
│      es.toml
│      fr.toml
│      it.toml
│      ja.toml
│      ko.toml
│      zh-cn.toml
│      zh-hk.toml
│      zh-mo.toml
│      zh-sg.toml
│      zh-tw.toml
│
├─images
│      screenshot.jpg
│      tn.jpg
│
├─layouts
│  │  404.html
│  │  robots.txt
│  │  sitemap.xml
│  │
│  ├─partials
│  │  │  nav.html
│  │  │
│  │  ├─function
│  │  │      currentMenuItem.html
│  │  │      getImageSlice.html
│  │  │      langCode.html
│  │  │
│  │  ├─head
│  │  │      favicon.html
│  │  │      link.html
│  │  │      meta.html
│  │  │      seo.html
│  │  │
│  │  └─plugin
│  │          analytics.html
│  │          script.html
│  │          style.html
│  │
│  ├─shortcodes
│  │      year.html
│  │
│  └─_default
│          baseof.html
│          single.html
│          single.json
│
├─public
│  │  404.html
│  │  dot.png
│  │  dot.svg
│  │  go.mod
│  │  index.html
│  │  index.xml
│  │  robots.txt
│  │  sitemap.xml
│  │
│  ├─bundled
│  │  ├─css
│  │  │      main.css
│  │  │
│  │  └─js
│  │          B3pkGT.js
│  │          BSNAnE.js
│  │          CqsBem.js
│  │          EYMnLv.js
│  │          main.js
│  │
│  ├─categories
│  │      index.xml
│  │
│  ├─erwitt
│  │      index.html
│  │
│  ├─Info
│  │      index.html
│  │
│  ├─lib
│  │  └─fonts
│  │          fw.woff2
│  │          GeistVF.woff2
│  │          NotoSans-Regular.woff2
│  │          NotoSansCJKjp-Regular.woff2
│  │          NotoSansCJKkr-Regular.woff2
│  │          NotoSansCJKsc-Regular.woff2
│  │          NotoSansCJKtc-Regular.woff2
│  │
│  ├─scss
│  │      critical.css.map
│  │      critical.min.css
│  │
│  ├─tags
│  │      index.xml
│  │
│  └─webb
│          index.html
│
├─resources
│  └─_gen
│      └─assets
│          └─scss
│                  critical.scss_05a8f21199d8496725106db9f4052a50.content
│                  critical.scss_05a8f21199d8496725106db9f4052a50.json
│
├─static
│  │  dot.png
│  │  dot.svg
│  │
│  ├─bundled
│  │  ├─css
│  │  │      main.css
│  │  │
│  │  └─js
│  │          B3pkGT.js
│  │          BSNAnE.js
│  │          CqsBem.js
│  │          EYMnLv.js
│  │          main.js
│  │
│  └─lib
│      └─fonts
│              fw.woff2
│              GeistVF.woff2
│              NotoSans-Regular.woff2
│              NotoSansCJKjp-Regular.woff2
│              NotoSansCJKkr-Regular.woff2
│              NotoSansCJKsc-Regular.woff2
│              NotoSansCJKtc-Regular.woff2
│
├─themes
│  └─bridget
│      └─layouts
│          └─shortcodes
│                  year.html
│
└─_default
        hugo.toml
        markup.toml
        outputs.toml
        params.toml
        sitemap.toml
```
2. The config: 
```
# timeout
timeout = "1200s"
# your website url
baseURL = 'https://bridget-demo.sped0n.com'
# website title
title = 'Bridget'
# don't touch this
disableKinds = ["section", "taxonomy", "term", "home"]
# robots.txt
enableRobotsTXT = true
# available options
# * en (powered by Geist)
# * de (powered by Geist)
# * es (powered by Geist)
# * fr (powered by Geist)
# * it (powered by Geist)
# * zh-sg zh-cn (powered by Noto Sans SC)
# * zh-hk zh-tw zh-mo (powered by Noto Sans TC)
# * ja (powered by Noto Sans JP)
# * ko (powered by Noto Sans KR)
defaultContentLanguage = 'en'
markup.goldmark.renderer.unsafe = true

# theme as module
[module]
replacements = "github.com/Sped0n/bridget/v2 -> ../.." # deploy with local dir (relative to hugo site theme dir)  WARN: delete this line if you want to deploy with git
[[module.imports]]
path = "github.com/Sped0n/bridget/v2" # deploy with git (recommended)  WARN: you should also set `bundled` to true in params.toml !!!
```

ps: Currently, I am experiencing an issue where the template images are not displaying during a local demonstration.

I hope you can help me with this issue, thank you.

**Reply by 713b76b7-2247-4bd5-98a7-014226363f9b** [reply]
Do create a new site with hugo's new command and follow documentation, rather than messing with the project files directly

**Reply by Tomp-Akoe** [reply]
> Do create a new site with hugo's new command and follow documentation, rather than messing with the project files directly

Hold on, let me try again because after following the documentation, I encountered a "Page not found" error.

**Reply by Tomp-Akoe** [reply]
> Do create a new site with hugo's new command and follow documentation, rather than messing with the project files directly


Hello, I have reconfigured, but encountered the following errors:

1. After running hugo server:
```
Watching for changes in E:\tools\hugo_0.140.0_windows-amd64\second\{archetypes,assets,content,data,i18n,layouts,static}
Watching for config changes in E:\tools\hugo_0.140.0_windows-amd64\second\hugo.toml, E:\tools\hugo_0.140.0_windows-amd64\second\config\_default
Start building sites …
hugo v0.140.0-3f35721fb2c75a1f7cc5a7a14400b66e73d4b06e+extended windows/amd64 BuildDate=2024-12-17T14:20:55Z VendorInfo=gohugoio

Built in 1 ms
Error: error building site: process: readAndProcessContent: "E:\tools\hugo_0.140.0_windows-amd64\second\content\Info\index.md:25:8": failed to extract shortcode: template for shortcode "year" not found
```

2. Detailed list of the reconfigured project folder.
```
│  .hugo_build.lock
│  hugo.toml
│  markup.toml
│  outputs.toml
│  params.toml
│  sitemap.toml
│
├─archetypes
│      default.md
│
├─assets
├─config
│  └─_default
│          hugo.toml
│          markup.toml
│          outputs.toml
│          params.toml
│          sitemap.toml
│
├─content
│  ├─Erwitt
│  │      1.jpg
│  │      10.jpg
│  │      11.jpg
│  │      12.jpg
│  │      13.jpg
│  │      14.jpg
│  │      15.jpg
│  │      16.jpg
│  │      17.jpg
│  │      18.jpg
│  │      19.jpg
│  │      2.jpg
│  │      20.jpg
│  │      21.jpg
│  │      22.jpg
│  │      23.jpg
│  │      24.jpg
│  │      25.jpg
│  │      3.jpg
│  │      4.jpg
│  │      5.jpg
│  │      6.jpg
│  │      7.jpg
│  │      8.jpg
│  │      9.jpg
│  │      index.md
│  │
│  ├─Gruyaert
│  │      1.jpg
│  │      10.jpg
│  │      11.jpg
│  │      12.jpg
│  │      13.jpg
│  │      14.jpg
│  │      15.jpg
│  │      16.jpg
│  │      17.jpg
│  │      18.jpg
│  │      19.jpg
│  │      2.jpg
│  │      20.jpg
│  │      21.jpg
│  │      22.jpg
│  │      23.jpg
│  │      24.jpg
│  │      25.jpg
│  │      26.jpg
│  │      27.jpg
│  │      28.jpg
│  │      29.jpg
│  │      3.jpg
│  │      30.jpg
│  │      31.jpg
│  │      32.jpg
│  │      33.jpg
│  │      34.jpg
│  │      35.jpg
│  │      4.jpg
│  │      5.jpg
│  │      6.jpg
│  │      7.jpg
│  │      8.jpg
│  │      9.jpg
│  │      index.md
│  │
│  ├─Info
│  │      index.md
│  │
│  └─Webb
│          1.jpg
│          10.jpg
│          11.jpg
│          12.jpg
│          13.jpg
│          14.jpg
│          15.jpg
│          16.jpg
│          17.jpg
│          18.jpg
│          19.jpg
│          2.jpg
│          20.jpg
│          21.jpg
│          22.jpg
│          23.jpg
│          24.jpg
│          25.jpg
│          26.jpg
│          27.jpg
│          28.jpg
│          29.jpg
│          3.jpg
│          30.jpg
│          4.jpg
│          5.jpg
│          6.jpg
│          7.jpg
│          8.jpg
│          9.jpg
│          index.md
│
├─data
├─i18n
├─layouts
├─public
│  │  dot.png
│  │  dot.svg
│  │  index.xml
│  │  sitemap.xml
│  │
│  ├─categories
│  │      index.xml
│  │
│  └─tags
│          index.xml
│
├─static
└─themes
    └─bridget
        │  .eslintignore
        │  .eslintrc.json
        │  .gitignore
        │  .prettierignore
        │  .prettierrc.json
        │  CODE_OF_CONDUCT.md
        │  go.mod
        │  LICENSE
        │  package.json
        │  pnpm-lock.yaml
        │  README.md
        │  theme.toml
        │  tsconfig.json
        │  vite.config.ts
        │
        ├─.github
        │  │  codeql-config.yml
        │  │  dependabot.yml
        │  │
        │  ├─ISSUE_TEMPLATE
        │  │      bug_report.yaml
        │  │      config.yml
        │  │      feature_request.yaml
        │  │
        │  └─workflows
        │          build.yml
        │          codeql.yml
        │          eslint.yml
        │
        ├─archetypes
        │      default.md
        │
        ├─assets
        │  ├─scss
        │  │  │  critical.scss
        │  │  │  style.scss
        │  │  │  _variables.scss
        │  │  │
        │  │  ├─_core
        │  │  │      _base.scss
        │  │  │      _font.scss
        │  │  │      _mixins.scss
        │  │  │      _reset.scss
        │  │  │      _typography.scss
        │  │  │
        │  │  └─_partial
        │  │          _article.scss
        │  │          _collection.scss
        │  │          _container.scss
        │  │          _customCursor.scss
        │  │          _gallery.scss
        │  │          _nav.scss
        │  │          _stage.scss
        │  │          _stageNav.scss
        │  │
        │  └─ts
        │      │  main.tsx
        │      │  resources.ts
        │      │  state.tsx
        │      │  utils.ts
        │      │
        │      ├─desktop
        │      │      customCursor.tsx
        │      │      layout.tsx
        │      │      nav.tsx
        │      │      stage.tsx
        │      │      stageNav.tsx
        │      │
        │      └─mobile
        │              collection.tsx
        │              gallery.tsx
        │              galleryImage.tsx
        │              galleryNav.tsx
        │              layout.tsx
        │
        ├─doc
        │      getStarted.md
        │
        ├─exampleSite
        │  │  go.mod
        │  │
        │  ├─config
        │  │  └─_default
        │  │          hugo.toml
        │  │          markup.toml
        │  │          outputs.toml
        │  │          params.toml
        │  │          sitemap.toml
        │  │
        │  ├─content
        │  │  ├─Erwitt
        │  │  │      1.jpg
        │  │  │      10.jpg
        │  │  │      11.jpg
        │  │  │      12.jpg
        │  │  │      13.jpg
        │  │  │      14.jpg
        │  │  │      15.jpg
        │  │  │      16.jpg
        │  │  │      17.jpg
        │  │  │      18.jpg
        │  │  │      19.jpg
        │  │  │      2.jpg
        │  │  │      20.jpg
        │  │  │      21.jpg
        │  │  │      22.jpg
        │  │  │      23.jpg
        │  │  │      24.jpg
        │  │  │      25.jpg
        │  │  │      3.jpg
        │  │  │      4.jpg
        │  │  │      5.jpg
        │  │  │      6.jpg
        │  │  │      7.jpg
        │  │  │      8.jpg
        │  │  │      9.jpg
        │  │  │      index.md
        │  │  │
        │  │  ├─Gruyaert
        │  │  │      1.jpg
        │  │  │      10.jpg
        │  │  │      11.jpg
        │  │  │      12.jpg
        │  │  │      13.jpg
        │  │  │      14.jpg
        │  │  │      15.jpg
        │  │  │      16.jpg
        │  │  │      17.jpg
        │  │  │      18.jpg
        │  │  │      19.jpg
        │  │  │      2.jpg
        │  │  │      20.jpg
        │  │  │      21.jpg
        │  │  │      22.jpg
        │  │  │      23.jpg
        │  │  │      24.jpg
        │  │  │      25.jpg
        │  │  │      26.jpg
        │  │  │      27.jpg
        │  │  │      28.jpg
        │  │  │      29.jpg
        │  │  │      3.jpg
        │  │  │      30.jpg
        │  │  │      31.jpg
        │  │  │      32.jpg
        │  │  │      33.jpg
        │  │  │      34.jpg
        │  │  │      35.jpg
        │  │  │      4.jpg
        │  │  │      5.jpg
        │  │  │      6.jpg
        │  │  │      7.jpg
        │  │  │      8.jpg
        │  │  │      9.jpg
        │  │  │      index.md
        │  │  │
        │  │  ├─Info
        │  │  │      index.md
        │  │  │
        │  │  └─Webb
        │  │          1.jpg
        │  │          10.jpg
        │  │          11.jpg
        │  │          12.jpg
        │  │          13.jpg
        │  │          14.jpg
        │  │          15.jpg
        │  │          16.jpg
        │  │          17.jpg
        │  │          18.jpg
        │  │          19.jpg
        │  │          2.jpg
        │  │          20.jpg
        │  │          21.jpg
        │  │          22.jpg
        │  │          23.jpg
        │  │          24.jpg
        │  │          25.jpg
        │  │          26.jpg
        │  │          27.jpg
        │  │          28.jpg
        │  │          29.jpg
        │  │          3.jpg
        │  │          30.jpg
        │  │          4.jpg
        │  │          5.jpg
        │  │          6.jpg
        │  │          7.jpg
        │  │          8.jpg
        │  │          9.jpg
        │  │          index.md
        │  │
        │  └─static
        │          dot.png
        │          dot.svg
        │
        ├─i18n
        │      de.toml
        │      en.toml
        │      es.toml
        │      fr.toml
        │      it.toml
        │      ja.toml
        │      ko.toml
        │      zh-cn.toml
        │      zh-hk.toml
        │      zh-mo.toml
        │      zh-sg.toml
        │      zh-tw.toml
        │
        ├─images
        │      screenshot.jpg
        │      tn.jpg
        │
        ├─layouts
        │  │  404.html
        │  │  robots.txt
        │  │  sitemap.xml
        │  │
        │  ├─partials
        │  │  │  nav.html
        │  │  │
        │  │  ├─function
        │  │  │      currentMenuItem.html
        │  │  │      getImageSlice.html
        │  │  │      langCode.html
        │  │  │
        │  │  ├─head
        │  │  │      favicon.html
        │  │  │      link.html
        │  │  │      meta.html
        │  │  │      seo.html
        │  │  │
        │  │  └─plugin
        │  │          analytics.html
        │  │          script.html
        │  │          style.html
        │  │
        │  ├─shortcodes
        │  │      year.html
        │  │
        │  └─_default
        │          baseof.html
        │          single.html
        │          single.json
        │
        └─static
            ├─bundled
            │  ├─css
            │  │      main.css
            │  │
            │  └─js
            │          B3pkGT.js
            │          BSNAnE.js
            │          CqsBem.js
            │          EYMnLv.js
            │          main.js
            │
            └─lib
                └─fonts
                        fw.woff2
                        GeistVF.woff2
                        NotoSans-Regular.woff2
                        NotoSansCJKjp-Regular.woff2
                        NotoSansCJKkr-Regular.woff2
                        NotoSansCJKsc-Regular.woff2
                        NotoSansCJKtc-Regular.woff2
```

**Reply by 713b76b7-2247-4bd5-98a7-014226363f9b** [reply]
show me your `config/_default/hugo.toml`

**Reply by Tomp-Akoe** [reply]
> show me your `config/_default/hugo.toml`

```
# timeout
timeout = "1200s"
# your website url
baseURL = 'https://bridget-demo.sped0n.com'
# website title
title = 'Bridget'
# don't touch this
disableKinds = ["section", "taxonomy", "term", "home"]
# robots.txt
enableRobotsTXT = true
# available options
# * en (powered by Geist)
# * de (powered by Geist)
# * es (powered by Geist)
# * fr (powered by Geist)
# * it (powered by Geist)
# * zh-sg zh-cn (powered by Noto Sans SC)
# * zh-hk zh-tw zh-mo (powered by Noto Sans TC)
# * ja (powered by Noto Sans JP)
# * ko (powered by Noto Sans KR)
defaultContentLanguage = 'en'
markup.goldmark.renderer.unsafe = true

# theme as module
[module]
replacements = "github.com/Sped0n/bridget/v2 -> ../.." # deploy with local dir (relative to hugo site theme dir)  WARN: delete this line if you want to deploy with git
[[module.imports]]
path = "github.com/Sped0n/bridget/v2" # deploy with git (recommended)  WARN: you should also set `bundled` to true in params.toml !!!
```
Unmodified

**Reply by 713b76b7-2247-4bd5-98a7-014226363f9b** [reply, marked as answer]
Read the doc please, you should set it to `replacements = "github.com/Sped0n/bridget/v2 -> bridget"`

**Reply by Tomp-Akoe** [reply]
> 请阅读文档，您应该将其设置为`replacements = "github.com/Sped0n/bridget/v2 -> bridget"`

yep!!!!!!thank!!!!!
